README.md

```diff
@@ -134,3 +143,36 @@ More information are provided within the [ui-tests](./ui-tests/README.md) README
 ### Packaging the extension
 
 See [RELEASE](RELEASE.md)
+
+### Configuring the extension
+The server extension comes with some configurable options, that helps with replacing the Scheduler server API, replacing the execution engine, re-creating database tables, selecting a database path. Here is the list of all options.
+
+#### drop_tables
+Setting this value to `True` will re-create the database tables on each JupyterLab start.
```

Comment:
```suggestion
Setting this value to `True` will re-create the database tables on each JupyterLab start. This will destroy all existing data. It may be necessary if your database's schema is out of date.
```